src/sidebar/components/test/annotation-action-bar-test.js

```diff
@@ -276,26 +275,12 @@ describe('AnnotationActionBar', () => {
         button.props().onClick();
       });
 
-      assert.calledWith(fakeAnnotationMapper.flagAnnotation, fakeAnnotation);
-    });
-
-    it('updates flag state in store after flagging on service is successful', async () => {
```

Comment:
Updating the flag state in the store after calling the API is now done by `annotationsService.flag`.